x/evidence/client/rest/query.go

```diff
@@ -27,6 +27,8 @@ func registerQueryRoutes(clientCtx client.Context, r *mux.Router) {
 
 func queryEvidenceHandler(clientCtx client.Context) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
+		client.AddDeprecationHeaders(w)
```

Comment:
Neither of these evidence handlers have corresponding Swagger docs, should they?